library/src/commonMain/kotlin/com/lagradost/cloudstream3/extractors/InternetArchive.kt

```diff
@@ -0,0 +1,112 @@
+package com.lagradost.cloudstream3.extractors
+
+import com.lagradost.cloudstream3.SubtitleFile
+import com.lagradost.cloudstream3.mvvm.logError
+import com.lagradost.cloudstream3.utils.ExtractorApi
+import com.lagradost.cloudstream3.utils.ExtractorLink
+import com.lagradost.cloudstream3.utils.Qualities
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import java.net.URLDecoder
+
+open class InternetArchive : ExtractorApi() {
+    override val mainUrl = "https://archive.org"
```

Comment:
If archive.org has a stable video url scheme it would be beneficial to add a comment with an example video for future testing. Many already merged extractors are untestable due to unknown video urls. This is of course not required for merge.

Reply:
Done